healpy/newvisufunc.py

```diff
@@ -46,13 +53,23 @@ def mollview(
     flip="astro",
     format="%g",
     cbar=True,
-    cmap=None,
+    cmap='jet',
```

Comment:
we don't want to use jet, https://jakevdp.github.io/blog/2014/10/16/how-bad-is-your-colormap/, I would leave it to None so we use the matplotlib default colormap.

Reply:
I changed it to matplotlib default, but I had to assign it explicitly with "cmap=viridis", putting "cmap=None" does not go well with plot_surface